src/picongpu/submit/pizdaint-cscs/SLURM_Tutorial.md

```diff
@@ -0,0 +1,27 @@
+SLURM examples
```

Comment:
There are more differences `--cpus-per-task` is a not defined parameter. The concepts to allocate a node differ from the taurus-tud SLURM.
It is in general the same batch system but the handling is different in some parts. I can add a comment to the alias but keep this information also in this file.
IMO this short tutorial should be specific for Piz Daint.